setuptools/tests/test_sdist.py

```diff
@@ -449,6 +449,20 @@ def test_sdist_with_latin1_encoded_filename(self):
             except UnicodeDecodeError:
                 filename not in cmd.filelist.files
 
+    def test_pyproject_toml_in_sdist(self):
+        """
+        Check if pyproject.toml is included in source distribution if present
+        """
+        open(os.path.join(self.temp_dir, 'pyproject.toml'), 'w').close()
```

Comment:
We should maybe have a rough backport of `Pathlib.touch` like we have in [`pkg_resources`](https://github.com/pypa/setuptools/blob/e6bdf25f6ab5bf4d32b0f9affa0ab98ea35f3a29/pkg_resources/tests/test_pkg_resources.py#L249) for this purpose.

Not a big deal, but I don't love this style to create an empty file.

Reply:
Agreed, however doing so would imply that should be done on line 95 and 108 and others, so let's make that change in a separate effort.